runtime/events.go

```diff
@@ -0,0 +1,60 @@
+package runtime
```

Comment:
I thought runtime contained what is necessary for wiring an app.
As this is more for modules, does it still make sense to have it live here?